packages/terra-site/src/index.html

```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="utf-8">
     <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <meta http-equiv="Content-Language" content="en-US">
```

Comment:
What's the logic for setting this in the meta tag? This is set on the html element via the lang attribute.